### PR TITLE
[SD-2645] - Global Publish Filters

### PR DESCRIPTION
**Pull request by akintolga**

### CODE REVIEW

File: server/apps/publish/archive_publish.py [hunk 1]
```diff
@@ -479,6 +486,17 @@ def conforms_publish_filter(self, subscriber, doc):
         else:
             return publish_filter['filter_type'] == 'blocking'
 
+    def conforms_global_filter(self, subscriber, global_filters, doc):
+        service = get_resource_service('publish_filters')
```
Comment (sivakuna-aap) [inline server/apps/publish/archive_publish.py]: It might be better to have some documentation on both filter methods

**Comment by sivakuna-aap** [comment]
Needs rebasing

**Comment by sivakuna-aap** [comment]
:100:

**Comment by mdhaman** [comment]
:+1: :100:

**Comment by amagdas** [comment]
:+1:

**Pull request merged**